code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -188,24 +188,30 @@
 		msg += "[t_He] [p_do()]n't seem all there.\n"
 
 	if(temp)
-		if(temp < 30)
+		if(temp < 25)
 			msg += "[t_He] [t_has] minor bruising.\n"
+		else if (temp < 50)

```

Comment:
Space between the `else if` and the `(`

Not good.

Reply:
In tg's Contributing.md we don't care about this